nc_time_axis/__init__.py

```diff
@@ -233,14 +248,20 @@ def convert(cls, value, unit, axis):
                 return value
             first_value = value
 
-        if not hasattr(first_value, 'calendar'):
-            raise ValueError('A "calendar" attribute must be attached to '
-                             'netcdftime object to understand them properly.')
-        return netcdftime.date2num(value, cls.standard_unit,
-                                   calendar=first_value.calendar)
+        if not isinstance(first_value.datetime, netcdftime.datetime):
+            raise ValueError('The datetime attribute of the CalendarDateTime '
+                             'object must be of type `netcdftime.datetime`.')
+
+        ut = netcdftime.utime(cls.standard_unit, calendar=first_value.calendar)
+
+        if isinstance(value, Iterable):
+            num = ut.date2num([v.datetime for v in value])
+        else:
+            num = ut.date2num(value.datetime)
```

Comment:
So... looking at matplotlib ...
In matplotlib/axis.py, matplotlib makes use of this `convert` method to convert the tick values and to set the default intervals. In the setting of the default intervals, it is passing a single element to the `convert` method and uses the result to set the `axes.dataLim.intervalx` and `axes.viewLim.intervalx` (and the same for y).

I would think it would be a problem in places like [here](https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/axes/_axes.py#L2129)... but it doesn't seem to be  

When testing the values that come out, for normal numbers you get:
```
>>> plt.plot([1, 2, 3])
[<matplotlib.lines.Line2D object at 0x24d9510>]
>>> plt.gca().viewLim.intervaly
array([ 1.,  3.])
```

for netcdftime.datetimes:
```
>>> dates = [nc_time_axis.CalendarDateTime(netcdftime.datetime(1986, month, 30), '360_day') for month in range(1, 6)]
>>> line1, = plt.plot(dates)
>>> plt.gca().viewLim.intervaly
array([ -5.01100000e+03,   3.00000000e+00])
```

Basically it doesn't seem to be a problem